apps/modernization-ui/src/components/Accordion/accordion.module.scss

```diff
@@ -0,0 +1,46 @@
+@use 'styles/colors';
+
+.accordian {
+    background-color: colors.$base-lightest;
+    summary {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        color: colors.$base-darkest;
+        padding: 1rem;
+        cursor: pointer;
+        &::marker {
+            content: "";
+        }
+        h3 {
+            margin: 0;
+            font-size: 1rem;
+        }
+        span {
+            background-size: 8px 24px;
+            display: flex;
+            width: 0.875rem;
+            height: 0.125rem;
+            content: "";
+            background-color: colors.$base-darkest;
+        }
+    }
+    .content {
+        background-color: colors.$base-white;
+        padding: 0 1rem;
+        label {
+            margin-top: 0;
+            padding-top: 1rem;
+        }
+        input {
+            width: calc(100% - 1rem);
+            height: 1.5rem;
+            margin-bottom: 0.5rem;
+        }
+        select {
+            width: calc(100% - 2.5rem);
+            height: 1.5rem;
+            margin-bottom: 0.5rem;
+        }
```

Comment:
(non-blocker) ideally the accordion would not have special rules for content.  I wonder what is causing the `input` or `select` elements to render so tall.

Reply:
USWDS Inputs and Select have all kinds of padding and margins -- a lot of times they're placeholders for error messages.